server/services/stadiums.ts

```diff
@@ -0,0 +1,27 @@
+import { Request } from 'express';
+import { Gallery, Stadium, User } from '../models';
+
+export const getAllStadiumsService = async (
+  req: Request,
+): Promise<object | undefined> => {
+  const res = await User.findAll({
+    where: { role: 'STADIUM' },
+    attributes: ['username', 'id'],
+  });
+  return res;
```

Comment:
handle no stadiums Case, 
make Sure that you send the right Status code, as you made in the create match Service